documentation/modules/cruise-control/con-rebalance-performance.adoc

```diff
@@ -44,6 +44,15 @@ These strategies can be configured as a sequence.
 The first strategy attempts to compare two partition reassignments using its internal logic. 
 If the reassignments are equivalent, then it passes them to the next strategy in the sequence to decide the order, and so on.
 
+[discrete]
+== Intra-broker disk balancing
+
+Moving a large amount of data between disks on the same broker has less impact than between separate brokers
+If you are running a Kafka deployment that uses JBOD storage, Cruise Control can balance partitions between disks on the same broker. 
+To preform an intra-broker disk balance and only an intra-broker disk balance, set `rebalanceDisk` to `true` under the `KafkaRebalance.spec`.
```

Comment:
typo "perform"

Reply:
Other than the typo, I didn't get what the sentence mean ... is there a repetition?

"To preform an intra-broker disk balance and only an intra-broker disk balance" ... what do you mean?

Reply:
I meant to imply that there is no inter-broker balance happening when an intra-broker balance it preformed but it was not clear so I removed it!